### PR TITLE
Javascript/javascript1/week4

### PR DESCRIPTION
**Pull request by Hani-far**

### CODE REVIEW

File: javascript/javascript1/week4/voiceAssistant.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+// Voice assistant
+
+let userName = "";
+let toDo = [];
+
+function getReply(command) {
+
+    if (typeof command !== "string") {
+        return "Please retype your Question";
```
Comment (cameronjdavis) [inline javascript/javascript1/week4/voiceAssistant.js]: Give an error message that says what is wrong, not just that something is wrong.
```suggestion
        return "Your question is not a string, please enter a question that is a string.";
```

File: javascript/javascript1/week4/voiceAssistant.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    }
+    let str = command.split(" ");
```
Comment (cameronjdavis) [inline javascript/javascript1/week4/voiceAssistant.js]: `command.split()` returns an array, so the name `str` is confusing. Use variable names that explain what is in the variable. E.g. splitString, stringPieces.
```suggestion
    let stringPieces = command.split(" ");
```

File: javascript/javascript1/week4/voiceAssistant.js [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+        case "*": return a * b;
+
+            break;
+
+        case "/": return a / b;
+
+            break;
+
+    }
```
Comment (cameronjdavis) [inline javascript/javascript1/week4/voiceAssistant.js:91]: What if `math` is an unknown operator like `math = $`? Add a `default` case to the switch statement to handle all math operators that are not know. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch

**Review by cameronjdavis — CHANGES_REQUESTED** [review]

File: javascript/javascript1/week4/voiceAssistant.js (hunk 1 above)
Comment (Hani-far) [reply javascript/javascript1/week4/voiceAssistant.js]: Thanks for the advice.

File: javascript/javascript1/week4/voiceAssistant.js (hunk 2 above)
Comment (Hani-far) [reply javascript/javascript1/week4/voiceAssistant.js]: done.thanks

**Pull request merged**